test/seed.js

```diff
@@ -201,13 +201,13 @@ describe("Contract: Seed", async () => {
           expect((await setup.seed.softCap()).toString()).to.equal(
               softCap.toString()
           );
-          expect((await setup.seed.price()).toString()).to.equal(
-              price.toString()
-          );
+          // expect((await setup.seed.price()).toString()).to.equal(
```

Comment:
same

Reply:
Deleted all mentioned comments (all of the comments with deleted now parts)